openlibrary/plugins/openlibrary/js/edit.js

```diff
@@ -15,6 +15,18 @@ function update_len() {
     $('#excerpts-excerpt-len').html(2000 - len).css('color', color);
 }
 
+export function initEditRow(){
+    document.querySelector('#add_row_button').addEventListener('click', ()=>add_row('website'));
+}
+
+function add_row(name) {
+    const inputs = document.querySelector(`#clone_${name} #inputs`);
+    const cloned = inputs.children[0].cloneNode(true);
```

Comment:
It seems like we clone the node, just to avoid having to set the type attribute to text. How about using document.createElement instead?